tests/basic/basic_test.go

```diff
@@ -52,19 +52,21 @@ var _ = Describe("{VolumeDriverDown}", func() {
 			contexts = append(contexts, ScheduleAndValidate(fmt.Sprintf("voldriverdown-%d", i))...)
 		}
 
-		Step("get nodes for all apps in test and bounce volume driver", func() {
-			for _, ctx := range contexts {
-				appNodes := GetNodesThatCanBeDown(ctx)
+		Step("get nodes bounce volume driver", func() {
+			for _, appNode := range node.GetWorkerNodes() {
 				Step(
-					fmt.Sprintf("stop volume driver %s on app %s's nodes: %v",
-						Inst().V.String(), ctx.App.Key, appNodes),
+					fmt.Sprintf("stop volume driver %s on node: %s",
+						Inst().V.String(), appNode.Name),
 					func() {
-						StopVolDriverAndWait(appNodes)
+						StopVolDriverAndWait([]node.Node{appNode})
 					})
 
-				Step("starting volume driver", func() {
-					StartVolDriverAndWait(appNodes)
-				})
+				Step(
+					fmt.Sprintf("starting volume %s driver on node %s",
+						Inst().V.String(), appNode.Name),
+					func() {
+						StartVolDriverAndWait([]node.Node{appNode})
+					})
 
 				Step("Giving few seconds for volume driver to stabilize", func() {
 					time.Sleep(20 * time.Second)
```

Comment:
add validate context after this..

after bouncing PX on each node, we want to ensure all apps are fine.